sound/soc/sof/intel/Kconfig

```diff
@@ -91,6 +99,7 @@ config SND_SOC_SOF_BROADWELL_SUPPORT
 config SND_SOC_SOF_BROADWELL
 	tristate
 	select SND_SOC_SOF_INTEL_COMMON
+	select SND_SOC_SOF_INTEL_COMMON_IPC
```

Comment:
it's odd that SND_SOC_SOF_INTEL_COMMON_IPC is only used for legacy platforms before SKL. Probably needs to be renamed.

Reply:
Yes, that's for non-HDA platforms, I'm open for better names :-)

Reply:
SND_SOC_SOF_INTEL_HIFI_EP_IPC?

Reply:
HIFI_EP is the name of the Tensilica core used before SKL.